packages/design-tokens/src/db.ts

```diff
@@ -42,9 +47,22 @@ export const clone = async (
 };
 
 export const patch = async (
-  buildId: DbDesignTokens["buildId"],
-  patches: Array<Patch>
+  {
```

Comment:
design-tokens package will be removed. Already a PR. You can revert to avoid conflicts.

Reply:
already removed